### PR TITLE
Feature/handle existing pods creation

### PR DESCRIPTION
**Pull request by maheshrayas**
@jbrekelmans @xUnholy  #51 
Fixed couple of issues:
1. Don't error when the pod status is completed
2. Don't error when the pod already exists.

### CODE REVIEW

File: pkg/up/up.go [hunk 1]
```diff
@@ -535,14 +540,17 @@ func parsePodStatus(pod *v1.Pod) (podStatus, error) {
...
+				return podStatusOther, fmt.Errorf("aborting because container %s of pod %s terminated (code=%d,signal=%d,reason=%s): %s",
+					containerStatus.Name,
+					pod.ObjectMeta.Name,
+					t.ExitCode,
+					t.Signal,
+					t.Reason,
+					t.Message,
+				)
+			}
+			return 3, nil
```
Comment (jbrekelmans) [inline pkg/up/up.go]: Can use `podStatusCompleted` instead of `3`

File: pkg/up/up.go [hunk 2]
```diff
@@ -715,20 +731,25 @@ func (u *upRunner) run() error {
...
 		}
 	}
 	fmt.Printf("pods ready (%d/%d)\n", len(u.appsThatNeedToBeReady), len(u.appsThatNeedToBeReady))
 	return nil
 }
 
+func (u *upRunner) checkIfPodsReady() bool{
+	allPodsReady := true
+	for app := range u.appsThatNeedToBeReady {
+		if !(app.maxObservedPodStatus == podStatusReady || app.maxObservedPodStatus == podStatusCompleted)  {
```
Comment (jbrekelmans) [inline pkg/up/up.go]: Can simplify condition to `app. maxObservedPodStatus < podStatusReady `

File: pkg/up/up.go (hunk 1 above)
Comment (maheshrayas) [reply pkg/up/up.go]: Fixed

File: pkg/up/up.go (hunk 2 above)
Comment (maheshrayas) [reply pkg/up/up.go]: Fixed

**Pull request merged**